### PR TITLE
added dependencies

### PR DESCRIPTION
**Pull request by friedhelm-h**
I added some dependencies that were missing when I tried to compile the ROS package

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,11 @@ Other ROS dependencies should have been installed in Section 1.1.
 If not by accident, install the missing ones accordingly.
 Besides, you also need to have `OpenCV` (3.2 or later) and `Eigen 3` installed.
 
+Furthermore, `GSL` needs to be installed. If missing, you can install it with
```
Comment (guillermogb) [inline README.md:82]: OK

File: dependencies.yaml [hunk 2]
```diff
@@ -22,4 +22,12 @@ repositories:
   eigen_checks:
     type: git
     url: https://github.com/ethz-asl/eigen_checks.git
+    version: master
+  minkindr_ros:
```
Comment (guillermogb) [inline dependencies.yaml:26]: I think we do not need minkindr... this is supposed to be for 3D transformations, and I do not think we use any now. Could you indicate where is it used? @friedhelm-h  thanks.

Comment (friedhelm-h) [reply dependencies.yaml:26]: It was listed in the package.xml and resulted in a compile error this is why I thought it is needed.

Comment (guillermogb) [reply dependencies.yaml:26]: We still the GSL dependency in the README, correct? @friedhelm-h

Comment (guillermogb) [reply dependencies.yaml:26]: done! Thanks @friedhelm-h

**Pull request closed without merging**